### PR TITLE
refactor: enhancement UI/UX

### PR DESCRIPTION
**Pull request by phucvarmeta**

### CODE REVIEW

File: packages/web/src/components/incentivize/incentivize-block-tooltip-content/IncentivizeBlockTooptipContent.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    switch (period) {
+      case 180:
+        return 15552000;
+      case 365:
+        return 31536000;
+      case 90:
+      default:
+        return 7776000;
+    }
+  }, [period]);
```
Comment (jinoosss) [inline packages/web/src/components/incentivize/incentivize-block-tooltip-content/IncentivizeBlockTooptipContent.tsx:20]: Is it the number of blocks created by date?
I think you calculated the block generation time as 1 second, is that correct ?

Also, unless you're looking for the average block creation time, it seems like a good idea to keep the block average production time static.

Comment (phucvarmeta) [reply packages/web/src/components/incentivize/incentivize-block-tooltip-content/IncentivizeBlockTooptipContent.tsx:20]: image

File: packages/web/src/hooks/swap/use-swap-handler.tsx [hunk 2]
```diff
@@ -385,17 +402,22 @@ export const useSwapHandler = () => {
   }, [swapButtonState]);
 
   const swapTokenInfo: SwapTokenInfo = useMemo(() => {
+    console.log(
+      "🚀 ~ constswapTokenInfo:SwapTokenInfo=useMemo ~ tokenBUSD:",
+      tokenBUSD,
+    );
```
Comment (jinoosss) [inline packages/web/src/hooks/swap/use-swap-handler.tsx:408]: Please remove console log.

File: packages/web/src/utils/new-number-utils.ts [hunk 3]
```diff
@@ -0,0 +1,262 @@
...
+  //   : null;
+
+  // if (
+  //   hasMinLimit &&
+  //   internalMinLimit &&
+  //   valueAsBigNum.isLessThan(internalMinLimit) &&
+  //   valueAsBigNum.isGreaterThan(0)
+  // ) {
+  //   return `<${prefix}${internalMinLimit}`;
+  // }
```
Comment (jinoosss) [inline packages/web/src/utils/new-number-utils.ts:180]: Please remove unused lines.

File: packages/web/src/utils/number-utils.ts [hunk 4]
```diff
@@ -711,17 +712,13 @@ export function subscriptFormat(
 ) {
   const numberStr = BigNumber(number).toFixed();
   const numberOfZero = countZeros(numberStr);
-  const significantDigits = options?.significantDigits || 5;
+  // const significantDigits = options?.significantDigits || 5;
```
Comment (jinoosss) [inline packages/web/src/utils/number-utils.ts:715]: Please remove unused lines.

File: packages/web/src/components/incentivize/pool-incentivize-details/PoolIncentivizeDetails.tsx [hunk 5]
```diff
@@ -10,8 +10,8 @@ import { getDateUtcToLocal } from "@common/utils/date-util";
 import dayjs from "dayjs";
 import { useGnotToGnot } from "@hooks/token/use-gnot-wugnot";
 import MissingLogo from "@components/common/missing-logo/MissingLogo";
-import Tooltip from "@components/common/tooltip/Tooltip";
-import IncentivizeBlockTooltipContent from "../incentivize-block-tooltip-content/IncentivizeBlockTooptipContent";
+// import Tooltip from "@components/common/tooltip/Tooltip";
+// import IncentivizeBlockTooltipContent from "../incentivize-block-tooltip-content/IncentivizeBlockTooptipContent";
```
Comment (jinoosss) [inline packages/web/src/components/incentivize/pool-incentivize-details/PoolIncentivizeDetails.tsx:14]: Please remove unused lines.

Comment (phucvarmeta) [reply packages/web/src/components/incentivize/pool-incentivize-details/PoolIncentivizeDetails.tsx:14]: I think should be leave it there if we need to comback with block calculation

**Review by jinoosss — APPROVED** [review]

**Review by AkydA — APPROVED** [review]

File: packages/web/src/components/incentivize/incentivize-block-tooltip-content/IncentivizeBlockTooptipContent.tsx (hunk 1 above)
Comment (phucvarmeta) [reply packages/web/src/components/incentivize/incentivize-block-tooltip-content/IncentivizeBlockTooptipContent.tsx]: Commented the code out for now

**Pull request merged**